backend/init_data/skills/pptx-slide/create_pptx.py

```
@@ -0,0 +1,826 @@
+
+        try:
+            # Import context service and database utilities
+            from app.db.session import SessionLocal
+            from app.services.context import context_service
+
+            # Create database session
+            db = SessionLocal()
+
+            try:
+                # Use context service to upload attachment
+                context, truncation_info = context_service.upload_attachment(
```

Ruff (0.14.11)

Unpacked variable `truncation_info` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)